src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -661,6 +665,16 @@ def load_arguments(self, _):
         c.argument('stdout', arg_type=bicep_stdout_type)
         c.argument('no_restore', arg_type=bicep_no_restore_type, help="When set, builds the bicep file without restoring external modules.")
 
+    with self.argument_context('bicep format') as c:
+        c.argument('file', arg_type=bicep_file_type, help="The path to the Bicep file to format in the file system.")
+        c.argument('outdir', arg_type=bicep_outdir_type)
+        c.argument('outfile', arg_type=bicep_outfile_type)
+        c.argument('stdout', arg_type=bicep_stdout_type)
+        c.argument('indent_kind', arg_type=bicep_indentkind_type, help="Set indentation kind. Valid values are ( Space | Tab ).")
+        c.argument('indent_size', arg_type=bicep_indentsize_type, help="Number of spaces to indent with (Only valid with --indent-kind set to Space).")
+        c.argument('insert_final_newline', arg_type=bicep_insertfinalnewline_type, help="Insert a final newline.")
+        c.argument('newline', arg_type=bicep_newline_type, help="Set newline char. Valid values are ( Auto | LF | CRLF | CR ).")
```

Comment:
Since the help message has been defined in `CLIArgumentType`, so there is no need to repeat the definition of help message
```suggestion
        c.argument('indent_kind', arg_type=bicep_indentkind_type)
        c.argument('indent_size', arg_type=bicep_indentsize_type)
        c.argument('insert_final_newline', arg_type=bicep_insertfinalnewline_type)
        c.argument('newline', arg_type=bicep_newline_type)
```